### ISSUE TITLE
Review and fix accessibility issues

### ISSUE THREAD

**Issue by tigerphilio93** [outline/outline#10106]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### This is not related to configuring Outline

- [x] The issue is not related to self-hosting config

### Current Behavior

Could we please review and (if applicable) fix the following accessibility issues found with various automated tools? For many industries, lack of full accessibility compliance could soon lead to legal liability.  The _number_ of issues may differ per Outline page, but the types of issues is likely the same on most Outline pages.  The list below does not cover any accessibility issues that can be found only from manual human testing.

Except for the last one listed, all of these accessibility checkers are freely available without cost:

- **axe browser browser extension**:
_Critical Issues: 6_
(1) Ensure an element's role supports its ARIA attributes
(5) Ensure buttons have discernible text
_Serious: 37_
(3) Ensure every ARIA input field has an accessible name
(18) Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
(1) Ensure links have discernible text
(15) Ensure interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies

- **WAVE browser extension**:
(15) Missing form label
(2) Language missing or invalid
(8) Empty button
(28) Very low contrast
(1) Suspicious alternative text
(1) Skipped heading level

- **Accessibility Insights browser extension**:
(1) aria-allowed-attr: Ensure an element's role supports its ARIA attributes
(3) aria-input-field-name: Ensure every ARIA input field has an accessible name
(6) button-name: Ensure buttons have discernible text
(20) color-contrast: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
(15) nested-interactive: Ensure interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies

- **AccessibilityChecker.org website**:
(52) Ensures all page content is contained by landmarks
(1) Ensure image alternative is not repeated as text
(1) Ensures the order of headings is semantically correct
(9) Ensure Buttons Have Accessible Names
(3) Ensure ARIA Attributes Are Allowed for Each Role
(3) Ensure ARIA Input Fields Have Accessible Names

### Environment

```markdown
- Outline:
- Browser:
```
